R/RTemplate.R

```diff
@@ -11,13 +19,10 @@ RT.standards = c(
   "  ret",
   "}",
   "if (!exists(\"include.dir\")) include.dir=NULL;",
+  "findFileInDirs = function(file, dirs)",
+  deparse(body(findFileInDirs)),
```

Comment:
I think you can `deparse(findFileInDirs)` the whole function (with arguments).

Reply:
Good idea